gunicorn/arbiter.py

```diff
@@ -584,6 +584,17 @@ def manage_workers(self):
                                   "value": active_worker_count,
                                   "mtype": "gauge"})
 
+        backlog = sum(
+            sock.get_backlog() or 0
+            for sock in self.LISTENERS
+        )
+
+        if backlog:
+            self.log.debug("socket backlog: {0}".format(backlog),
```

Comment:
I would imagine that different users may want to export this metric differently.

For some, reporting 0 backlog (as opposed to server not running) may be valuable.

Is there perhaps a precedent to register custom user hooks in this codebase?